packages/frontend/lib/congregation/congregation-information-board.ts

```diff
@@ -0,0 +1,45 @@
+import axios from "axios";
+
+import { backendRoutes } from "../config";
+import { requestOptions } from "../request-options";
+import {
+  InformationBoardItem,
+  informationBoardItemSchema,
+  NewInformationBoardItem,
+} from "../types/dashboard";
+
+export async function addInformationBoardItem(item: NewInformationBoardItem) {
+  try {
+    await axios.post(
+      backendRoutes.congregation.informationBoard.add,
+      item,
+      requestOptions(),
+    );
+  } catch (err) {
+    throw err;
+  }
+}
+
+/**
+ * Gets all information board items for current users' congregation
+ */
+export async function getCongInformationBoardItems(): Promise<
+  InformationBoardItem[]
+> {
+  const res = await axios.get(
+    backendRoutes.congregation.informationBoard.get,
+    requestOptions(),
+  );
+
+  const parsedData: InformationBoardItem[] = res.data.informationBoard.map(
+    (informationBoardItem: InformationBoardItem) => {
+      return informationBoardItem;
+
+      // return informationBoardItemSchema.parse(informationBoardItem);
+      // I dont know why this doesnt work
+      // At this point im too scared to find out why
+    },
+  );
+
+  return parsedData;
```

Comment:
Should take a look at this later

Reply:
Because if it didn't work, something is wrong with the schema itself, or somethings really wrong the the JSON our backer is returning.

Reply:
i just realised why... gorm or postgres adds extra ID, createdat and updatedat fields

Reply:
i was wrong

Reply:
there was a certain row in the table which had some messed up data from before